homeassistant/components/husqvarna_automower/switch.py

```diff
@@ -51,6 +51,11 @@ def is_on(self) -> bool:
             and attributes.planner.restricted_reason == RestrictedReasons.NOT_APPLICABLE
         )
 
+    @property
+    def available(self) -> bool:
+        """Return True if the device is available."""
+        return super().available and self.mower_attributes.metadata.connected
```

Comment:
Why is this added here? It's the same as in the mower entity? Shouldn't all entities have this property?

Reply:
My goal was to set it only for entities which are able to control something. Like the mower or the switch. I would like to make other entities like sensors or a device tracker available all the time. For example to find the mower if it's lost and not connected.
So I can add the available property to the base entity, and mark the other entities (which should always be available) as always available, or like suggested here.
As the standard value for `available` is `True` I choose this way.

Reply:
Add a new base class that has this property and inherits to the existing base class and let the mower and switch inherit the new base class.